tripleawarclub.org

```diff
@@ -1,9 +1,10 @@
 server {
     listen 80;
     listen [::]:80;
-    server_name tripleawarclub.org www.tripleawarclub.org;# dice.tripleawarclub.org *.tripleawarclub.org;
+    server_name tripleawarclub.org www.tripleawarclub.org;
```

Comment:
Let's encrypt now seems to support wildcard certificates.
So we could consider adding the wildcard back to this server name, if we decide to add the required DNS entries at some point.

Reply:
Note that there's already a wildcard entry on the HTTPS endpoint.  Should that be removed until we choose to support it across the board?

Reply:
Probably, yeah.
I missed that one.

Reply:
#25